packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -46,13 +46,21 @@ export class PlansPage {
 		this.page = page;
 	}
 
+	/**
+	 * Wait until the page is loaded and stable.
+	 */
+	async waitUntilLoaded(): Promise< void > {
```

Comment:
Nit: this can be private!

Reply:
Gotcha! Fixed in 9f224d1da43112f26d38162031c23444aa1adad6.